.github/workflows/bump-version.yml

```
@@ -0,0 +1,63 @@
+
+name: Bump version - wazuh-packages
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to bump to'
+        required: true
+      revision:
+        description: 'Revision to bump to. Defalut: 1'
+        required: false
+      date:
+        description: 'Date to bump to. Default: today'
+        required: false
+    
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Please try out `checkout@v3` as it uses node16.

> **[bump-version](https://github.com/wazuh/wazuh-packages/actions/runs/5543421039/jobs/10119402046)**
>The following actions uses node12 which is deprecated and will be forced to run on node16: actions/checkout@v2. For more info: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/
(https://github.com/wazuh/wazuh-packages/actions/runs/5543421039)


```suggestion
      - uses: actions/checkout@v3
```

More info: https://github.com/actions/checkout.

---

Done [71cbe5f](https://github.com/wazuh/wazuh-packages/pull/2270/commits/71cbe5f639f8c724526d715062d1e0cb3014e445)